master/internal/rm/actor_resource_manager.go

```diff
@@ -36,13 +37,36 @@ func (r *ActorResourceManager) ResolveResourcePool(
 	ctx actor.Messenger,
 	name string,
 	slots int,
-	command bool,
 ) (string, error) {
 	return name, nil
 }
 
+// ValidateResources is a default implementation to satisfy the interface, mostly for tests.
+func (r *ActorResourceManager) ValidateResources(
+	ctx actor.Messenger,
+	name string,
+	slots int,
+	command bool,
```

Comment:
can we remove this and collapse `ValidateResources` and `ValidateCommandResources` into one?

Reply:
Depends seems like this would involve updating everywhere that we check the relevant `sproto` i.e https://github.com/determined-ai/determined/blob/7832a93ef7666086cd9a2b185161dde498a07401/master/internal/rm/kubernetes_resource_manager.go#L253

If that sounds right and you think that refactor is worth it I can work on those updates. Also if you had some other type of refactor in mind let me know.